### PR TITLE
feat: New Camera Settings

### PR DESCRIPTION
**Pull request by Kinerius**
## What does this PR change?

- Added sensitivity setting (Debug panel)
- Added Noise setting (with value) (Debug panel) that only works for First person
- Added `V` hotkey to change from First to Third and vise versa
- Added `.` Hotkey to change Third person camera shoulder side
- Added F11 Hotkey to enter Free fly mode, press G to spawn the avatar over the new position
- Shoulder is forced to Center when the cursor is unlocked
- Removed zoom levels, now the V key and mouse Scroll will switch between First Person, Third Person, and Drone View
- Fixed camera transitions between First, Third, and Drone views.
- Added configurable camera aim offsets, this is noticeable when looking down in Third person, the character should not be centered, instead it should be slightly at the bottom of the screen.

## How to test the changes?

Check that the above changes work as expected.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: Explorer/Assets/DCL/Character/CharacterCamera/Systems/ApplyCinemachineSettingsSystem.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+using DCL.DebugUtilities.UIBindings;
+using ECS.Abstract;
+using System;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace DCL.Character.CharacterCamera.Systems
+{
+    [UpdateInGroup(typeof(CameraGroup))]
+    public partial class ApplyCinemachineSettingsSystem : BaseUnityLoopSystem
```
Comment (m3taphysics) [inline Explorer/Assets/DCL/Character/CharacterCamera/Systems/ApplyCinemachineSettingsSystem.cs:17]: Does this need to UpdateBefore in the camera group?

File: Explorer/Assets/DCL/Character/CharacterCamera/Systems/ApplyCinemachineSettingsSystem.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+        [Query]
+        private void UpdateCameraSettings(ref ICinemachinePreset cinemachinePreset)
+        {
+            float mMaxSpeed = currentSens / 100f;
+            cinemachinePreset.FirstPersonCameraData.POV.m_HorizontalAxis.m_MaxSpeed = mMaxSpeed;
+            cinemachinePreset.FirstPersonCameraData.POV.m_VerticalAxis.m_MaxSpeed = mMaxSpeed;
+            cinemachinePreset.FirstPersonCameraData.Noise.m_AmplitudeGain = cameraNoise ? noiseSlider.Value : 0;
+            cinemachinePreset.DroneViewCameraData.Camera.m_XAxis.m_MaxSpeed = mMaxSpeed;
+            cinemachinePreset.DroneViewCameraData.Camera.m_YAxis.m_MaxSpeed = mMaxSpeed / 100f;
```
Comment (m3taphysics) [inline Explorer/Assets/DCL/Character/CharacterCamera/Systems/ApplyCinemachineSettingsSystem.cs:63]: We should put this further division by 100 in a more descriptive temp as above too

File: Explorer/Assets/DCL/Character/CharacterMotion/Velocity/ApplyThirdPersonRotation.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+using UnityEngine;
+
+namespace DCL.Character.CharacterMotion.Velocity
+{
+    public static class ApplyThirdPersonRotation
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void Execute(ref CharacterRigidTransform rigidTransform, in MovementInputComponent input)
+        {
+            if (!(Mathf.Abs(input.Axes.x) > 0) && !(Mathf.Abs(input.Axes.y) > 0))
```
Comment (m3taphysics) [inline Explorer/Assets/DCL/Character/CharacterMotion/Velocity/ApplyThirdPersonRotation.cs:12]: Is it not simpler avoidng the two NOT's?

if (Mathf.Abs(input.Axes.x) < Mathf.Epsilon && Mathf.Abs(input.Axes.y) < Mathf.Epsilon)
    return;

**Review by m3taphysics — APPROVED** [review]
Approved with minors to look at!

File: Explorer/Assets/DCL/Character/CharacterCamera/Systems/ApplyCinemachineSettingsSystem.cs (hunk 1 above)
Comment (Kinerius) [reply Explorer/Assets/DCL/Character/CharacterCamera/Systems/ApplyCinemachineSettingsSystem.cs:17]: I thought about this but In the end I had no issues of this being executed before or after other systems

**Review by anicalbano — APPROVED** [review]
💚 Approved by QA 

https://github.com/decentraland/unity-explorer/assets/139157886/799af264-82f2-4ff6-aada-8ae91874ae40

**Pull request merged**